components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/pages/cdmf.page.devices/devices.js

```diff
@@ -23,6 +23,7 @@ function onRequest(context) {
 
     var groupName = request.getParameter("groupName");
     var groupOwner = request.getParameter("groupOwner");
```

Comment:
groupName is used in the same js as to change the title of the page. Other than that it is not used for any purpose. 
if (groupName) {
        title = groupName + " " + title;
        viewModel.groupName = groupName;
    }